lib/elixir/lib/option_parser.ex

```diff
@@ -173,18 +173,18 @@ defmodule OptionParser do
       {[debug: true], ["path/to/file"]}
 
       iex> OptionParser.parse!(["--limit", "xyz"], strict: [limit: :integer])
-      ** (OptionParser.ParseError) 1 error found!
-      --limit : Expected type integer, got "xyz"
+      ** (OptionParser.ParseError) found errors:
+      --limit - expected a value of type integer, got: "xyz"
```

Comment:
Honestly, I don't have a stronger opinion on uppercase or lowercase, so I
would keep it as is to avoid churn.

On Saturday, September 10, 2016, Andrea Leopardi <notifications@github.com>
wrote:

> In lib/elixir/lib/option_parser.ex
> <https://github.com/elixir-lang/elixir/pull/5202#discussion_r78275554>:
>
> > @@ -173,18 +173,18 @@ defmodule OptionParser do
> >        {[debug: true], ["path/to/file"]}
> >
> >        iex> OptionParser.parse!(["--limit", "xyz"], strict: [limit: :integer])
> > -      ** (OptionParser.ParseError) 1 error found!
> > -      --limit : Expected type integer, got "xyz"
> > +      ** (OptionParser.ParseError) found errors:
> > +      --limit - expected a value of type integer, got: "xyz"
>
> Oh, that's where it came from D: Any chance we can change at least the
> capital letter, as in --limit : expected ...? I mean in the Elixir
> command line as well. There may be reasons we do it like this but I don't
> know them, so pardon my ignorance here 😄
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/5202/files/2fcf09ec35407c560e4fbe670e34f3099dcec137#r78275554>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAlbqnkM7Cz9fa-zIjjZddURha0NkGOks5qoowngaJpZM4J5sU1>
> .
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*